frontend/packages/kubevirt-plugin/integration-tests/tests/models/kubevirtDetailView.ts

```diff
@@ -0,0 +1,73 @@
+/* eslint-disable no-await-in-loop */
```

Comment:
You can disable or tweak `no-await-in-loop` (and other rules too) in the appropriate `.eslintrc.js` file.